content/bn/devops.md

```diff
@@ -0,0 +1,16 @@
+---
+title: ডেভওপস (DevOps)
+status: Completed
+category: ধারণা
+---
+
+## এটা কি
+ডেভওপস হল একটি পদ্ধতি যেখানে দলগুলি অ্যাপ্লিকেশন ডেভেলপমেন্ট থেকে প্রোডাকশন অপারেশন পর্যন্ত সম্পূর্ণ প্রক্রিয়ার পরিচালনা করে থাকে। এটি সাধারণ প্রযুক্তি থেকে উচ্চ পর্যায় রয়েছে এবং সাধারণ ধরন থেকে আলাদা হয়। ডেভওপস প্রকৌশলীদের দলদের জন্য আহ্বান করে যারা ছোট উপাদানগুলিতে কাজ করে (একটি সম্পূর্ণ বৈশিষ্ট্যের বিপরীতে), হ্যান্ডঅফগুলি হ্রাস করে – যা সাধারণ ভুলের কারন।
+
+## এটি যেই সমস্যাটি নির্দেশ করে
+ঐতিহ্যগতভাবে, জটিল সংস্থা সাথে [tightly-coupled](/tightly_coupled_architectures/) [monolithic apps](/monolithic_apps/) , কাজ সাধারণত একাধিক দলের মধ্যে খণ্ডিত ছিল. এটি অসংখ্য হ্যান্ডঅফ এবং দীর্ঘ পরবর্তী সময়, প্রতিবার একটি উপাদান বা আপডেট প্রস্তুত ছিল, এটি পরবর্তী দলের জন্য একটি সারিতে স্থাপন করা হয়েছিল। যেহেতু ব্যক্তিরা কেবলমাত্র প্রকল্পের একটি ছোট অংশে কাজ করেছিল, এই পদ্ধতির ফলে মালিকানার অভাব দেখা দেয়। তাদের লক্ষ্য ছিল পরবর্তী দল কাছে কাজটি পৌঁছে দেওয়া, গ্রাহকের কাছে সঠিক কার্যকারিতা সরবরাহ না করা - অগ্রাধিকারগুলির একটি স্পষ্ট বিভ্রান্তি।
```

Comment:
Would it be better to use the Bengali words of "tightly-coupled" and "monolithic apps", rather than using the English words "tightly-coupled" and "monolithic apps" as-is?